C-Plus-Plus/cp/infix_to_postfix.cpp

```diff
@@ -0,0 +1,176 @@
+# Author: Prasad V Patil
+/*
+   C++ code for Conversion of infix expression to post fix expression.
+*/
+
+#include<iostream>
```

Comment:
Use #include <bits/stdc++.h> instead of adding several header files.

Reply:
@atarax665 replaced the header file.